projects/origin-release/bazel/Dockerfile

```diff
@@ -1,11 +1,55 @@
 # This Dockerfile is used to create origin-release:bazel necessary to build
 # openshift/installer smoke tests.
+# NOTE: This is the origin-release:golang-1.10 Dockerfile with bazel added.
+#
+# The standard name for this image is openshift/origin-release
+#
+FROM        centos:7
 
-FROM docker.io/openshift/origin-release:golang-1.10
+ENV VERSION=1.10.3 \
+    GOCACHE=/go/.cache \
+    GOARM=5 \
```

Comment:
Why do we need to set this?  Are we doing anything on ARM?

Reply:
> Why do we need to set this? Are we doing anything on ARM?

Ah, probably a copy/paste [here][1].  Is there a way we can DRY this up by depending on the `golang-1.10` tag locally without going through the docker.io registry?

[1]: https://github.com/openshift/release/blob/2b718e381245f70a9210962561d564074affbbed/projects/origin-release/golang-1.10/Dockerfile#L11

Reply:
yea, I copied/pasted bc I wanted it to be exactly `origin-release:golang-1.10 plus bazel`

Reply:
you can set the build config parent image (the source from)

Reply:
i don't know.. I hate that CI is broken at my expense atm though... and this origin-release:bazel tag is most likely only required short-term..

Reply:
oh, ok will set that in bc..